service/src/main/java/uk/gov/hmcts/reform/fpl/utils/AgeDisplayFormatHelper.java

```diff
@@ -0,0 +1,44 @@
+package uk.gov.hmcts.reform.fpl.utils;
+
+import java.time.Clock;
+import java.time.LocalDate;
+import java.time.Period;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+public class AgeDisplayFormatHelper {
+
+    private AgeDisplayFormatHelper() {
+    }
+
+    public static String formatAgeDisplay(final LocalDate dateOfBirth) {
+        checkNotNull(dateOfBirth, "Date of birth value is required");
+
+        LocalDate today = LocalDate.now(Clock.systemDefaultZone());
+
+        if (dateOfBirth.isAfter(today)) {
+            throw new IllegalArgumentException("Date of birth cannot be in the future");
+        }
```

Comment:
I don't think template logic should be doing validation like this, it should be caught in our validation logic for submitting a case i.e before this is generated